core/source/org/libreoffice/ide/eclipse/core/wizards/pages/NewUnoProjectPage.java

```diff
@@ -410,6 +411,19 @@ private void checkWhiteSpaces() {
         }
     }
 
+    /**
+     * Shows a error if there are spaces in the project name.
+     */
+    private void checkWhiteSpacesinProjectName() {
+
+        if (getProjectName().contains(" ")) { //$NON-NLS-1$
+            setMessage(Messages.getString("NewUnoProjectPage.WhiteSpacesInProjectNameError"), //$NON-NLS-1$
+                ERROR);
+            setPageComplete(false);
+        }
+
+    }
+
```

Comment:
These set of changes makes sure that an error sign is shown when there is a space given in the name of the Project and also the Next and Finish button are disabled.